packages/odyssey/src/scss/abstracts/_functions.scss

```diff
@@ -130,3 +130,17 @@ $icons: (
 
   @return url('data:image/svg+xml,#{$encoded}');
 }
+
+$z-index: (
+  top: 9999,
```

Comment:
Pretty wary of these numbers. Why not just 1, 2, 3...?

Also this how we did it in enduser and was very readable as well: https://github.com/okta/okta-ui-react/blob/master/packages/enduser/src/scss/abstracts/_variables.scss#L4

Reply:
Part of the rationale here is to account for existing or legacy UI and snowflakes. Without room to move between the system indexes, features won't be able to situate their own UI.

Carbon and SFLDS do something similar, and we felt like that was good evidence for scalability.

Reply:
In that case, shouldn't this be the driver to actually go ahead and fix them? Else we would just be stuck with this.

Reply:
Imagine that Odyssey ships components with `z-index`s of 1-10. When a feature team ships a snowflake that needs to sit between two of those values, they can't.

All UI devs won't be required to use this function to set their `z-index` (e.g. if someone needs to set something to `150`), but this is a way for Odyssey to stick to an internal standard that others can treat as static.

Reply:
My feeling is that if this is exposed from Odyssey, it should come with the assumption that it's going to get used and can be considered standard. 

If we are going to use this for only docs, we can move it to a doc-specific stylesheet. Are we sure we want these values to be used in components?

Reply:
Re: private functions -We will have functions and tokens that are Odyssey-only at some point, and they will have to ship with the system in order to be used by Odyssey. Engineers at all levels will need to learn how to use the system appropriately (and it's our job to document that).

The use case that mandated this change is limited to Modal + Docs, so I don't think this needs to ship right now, but the issue is practical in Odyssey today.

System Example: 

You have a Modal open and a Toast is triggered. What happens? Today that's determined by `position` properties and DOM order, but it should be determined by a system-internal z-standard.

Feature Example:

You have a Modal open and a Toast is triggered _and_ a Feature Component is also open or triggered. What happens? The only way for the Feature Component engineer to answer that question in all cases is by situating their `z-index` within one of the unused system indexes.

Reply:
I think this is fair. I have gone ahead and removed this from Odyssey and moved it in to the `vuepress-theme-odyssey` package, as well as tailored it to our docs site (for example, keys are now semantic `topbar`, `sidebar`, etc.)

Odyssey must provide guidance on how to do this, and the rush up to getting the docs site out isn't the time to think through and prescribe a solution to this problem. I logged issue #776 which outlines the work that (I believe) should be done as part of that effort.

Thanks for the feedback @santhoshbalakrishnan!